testutil-server/src/main/java/io/spine/testing/server/blackbox/SubscriptionFixture.java

```diff
@@ -0,0 +1,177 @@
+                SubscriptionService.newBuilder()
+                                   .add(context)
+                                   .build();
+        Observer updateObserver = new Observer(updates::add);
+        StreamObserver<Subscription> activator = new Activator(service, updateObserver);
+        service.subscribe(topic, activator);
+    }
+
+    /**
+     * Obtains a subject for asserting event messages received on the subscribed topic.
+     *
+     * <p>If messages of other kind were received instead, the returned iterable would be empty.
+     */
+    public IterableOfProtosSubject<EventMessage> assertEventMessages() {
+        ImmutableList<EventMessage> eventMessages =
+                updates.stream()
+                       .flatMap(SubscriptionFixture::toEventMessages)
+                       .collect(toImmutableList());
+        IterableOfProtosSubject<EventMessage> subject = ProtoTruth.assertThat(eventMessages);
+        return subject;
+    }
+
+    /**
+     * Obtains a subject for asserting entity states received on the subscribed topic.
+     *
+     * <p>If messages of other kind were received instead, the returned iterable would be empty.
+     */
+    public IterableOfProtosSubject<EntityState> assertEntityStates() {
+        ImmutableList<EntityState> states =
+                updates.stream()
+                       .flatMap(SubscriptionFixture::toEntityState)
+                       .collect(toImmutableList());
+        IterableOfProtosSubject<EntityState> subject = ProtoTruth.assertThat(states);
+        return subject;
+    }
+
+    private static Stream<EventMessage> toEventMessages(SubscriptionUpdate update) {
+        if (update.getUpdateCase() == EVENT_UPDATES) {
+            return update.eventMessages().stream();
+        }
+        return Stream.empty();
+    }
+
+    private static Stream<EntityState> toEntityState(SubscriptionUpdate update) {
+        if (update.getUpdateCase() == ENTITY_UPDATES) {
+            return update.states().stream();
+        }
+        return Stream.empty();
+    }
+
+    /**
+     * Counts the incoming subscription updates and feeds them to the given {@link Consumer}.
+     *
+     * <p>Re-throws all incoming errors as {@link IllegalStateException}.
+     */
+    private static final class Observer implements StreamObserver<SubscriptionUpdate> {
+
+        private final Consumer<SubscriptionUpdate> consumer;
+
+        private Observer(Consumer<SubscriptionUpdate> consumer) {
+            this.consumer = checkNotNull(consumer);
+        }
+
+        @Override
+        public void onNext(SubscriptionUpdate update) {
+            consumer.accept(update);
+        }
+
+        @Override
+        public void onError(Throwable t) {
+            throw new IllegalStateException(t);
+        }
+
+        @Override
+        public void onCompleted() {
+            // Do nothing.
+        }
+    }
+
+    /**
+     * Activates the first received {@link Subscription}.
```

Comment:
Judging on the implementation, it activates every subscription it receives. Not sure which one is the correct one — the implementation or the documentation.